resources/run_sampling.rb

```diff
@@ -306,38 +306,36 @@ def write_csv(sample_results, outfile)
   end
 end
 
-if __FILE__ == $PROGRAM_NAME
```

Comment:
It looks like a lot of changes, but all I did was remove this line (and un-indent everything within).